pkg/scheduler/controller/shoot/reconciler_test.go

```diff
@@ -89,6 +89,10 @@ var _ = Describe("Scheduler_Control", func() {
 						Type:   gardencorev1beta1.SeedBootstrapped,
 						Status: gardencorev1beta1.ConditionTrue,
 					},
+					{
```

Comment:
Can you add a test case similar to `should fail because it cannot find a seed cluster due to not bootstrapped` for the new condition?

Reply:
You will probably notice, that `common.VerifySeedReadiness` needs to be adapted to take the new condition into account.

Reply:
Nice catch, thanks!